collect_app/src/androidTest/java/org/odk/collect/android/feature/projects/GoogleDriveDeprecationTest.kt

```diff
@@ -47,21 +49,31 @@ class GoogleDriveDeprecationTest {
 
     @Test
     fun bannerIsVisibleInGoogleDriveProjects() {
-        CollectHelpers.addGDProject(gdProject1, "steph@curry.basket", testDependencies)
+        val newProject = CollectHelpers.addProject(Project.New("Old GD project", "A", "#ffffff"))
+        val component = DaggerUtils.getComponent(ApplicationProvider.getApplicationContext<Application>())
+        component
+            .settingsProvider()
+            .getMetaSettings()
+            .save(MetaKeys.getKeyIsOldGDProject(newProject.uuid), true)
 
         rule.startAtMainMenu()
             .openProjectSettingsDialog()
-            .selectProject(gdProject1.name)
+            .selectProject("Old GD project")
             .assertText(org.odk.collect.strings.R.string.google_drive_deprecation_message)
     }
 
     @Test
     fun forumThreadIsOpenedAfterClickingLearnMore() {
-        CollectHelpers.addGDProject(gdProject1, "steph@curry.basket", testDependencies)
+        val newProject = CollectHelpers.addProject(Project.New("Old GD project", "A", "#ffffff"))
```

Comment:
The project could probably be moved to shared private method here.

Reply:
Yes, done.